### PR TITLE
RELATED: BB-1694, DP-1563 MVF reveal + review, SSO

### PR DESCRIPTION
**Pull request by xMort**
Measure value filter documentation is revealed in the next version and reviewed by copywriter.

SSO documentation was updated.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/207/ : SUCCESS in 2m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/69/ : SUCCESS in 6s

File: docs/measure_value_filter_component.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
-<!--
-BB-1694 - Make this page visible by adding it to the menu and connect this article with measure_value_filter.md and vice versa
 ---
 id: measure_value_filter_component
 title: Measure Value Filter
 sidebar_label: Measure Value Filter
-copyright: (C) 2007-2018 GoodData Corporation
+copyright: (C) 2007-2020 GoodData Corporation
```
Comment (xMort) [inline docs/measure_value_filter_component.md]: Please review whole the file.

**Comment by xMort** [comment]
@evgeniabogranskaya could you please also review the following two files?
Unfortunately, you will not be able to write comments there. You would have to write the proposed changes here into the comments.

https://github.com/gooddata/gooddata-react-components/blob/develop/examples/src/routes/MeasureValueFilter.jsx

https://github.com/gooddata/gooddata-react-components/blob/develop/examples/src/routes/MeasureValueFilterComponent.jsx

File: docs/measure_value_filter_component.md (hunk 1 above)
Comment (evgeniabogranskaya) [reply docs/measure_value_filter_component.md]: Sure thing

**Comment by evgeniabogranskaya** [comment]
> @evgeniabogranskaya could you please also review the following two files?
> Unfortunately, you will not be able to write comments there. You would have to write the proposed changes here into the comments.
> 
> https://github.com/gooddata/gooddata-react-components/blob/develop/examples/src/routes/MeasureValueFilter.jsx
> 
> https://github.com/gooddata/gooddata-react-components/blob/develop/examples/src/routes/MeasureValueFilterComponent.jsx

I summarized changes in an email and sent it to @xMort .

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

File: website/i18n/en.json [hunk 2]
```diff
@@ -181,18 +181,16 @@
         "sidebar_label": "Line Chart"
       },
       "live_examples": {
-        "title": "_",
+        "title": "Live Examples",
```
Comment (xMort) [inline website/i18n/en.json]: This was autogenerated by the docusaurus.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/212/ : SUCCESS in 1m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/73/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/213/ : SUCCESS in 1m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/65/ : SUCCESS in 17s

**Pull request merged**